### DISCUSSION TITLE
Is it possible to make batch submissions?

### DISCUSSION THREAD

**Discussion by whitney** [CDCgov/seqsender#11] [General]
In the past our BFX team has made batch submissions to repositories like GISAID. Does seqsender support batch submissions to the other repositories via e.g. concatenating the FASTAs? If so how do we handle the batch submission of FASTQ files to SRA? Can we specify a value of "cloud" in the `SRA_file_location`, and then just provide an s3 _directory_ for the `SRA_file_column1` and `SRA_file_column2` fields?

**Comment by dthoward96** [comment]
Yes, SeqSender by default supports batch submissions. SeqSender expects all samples to be concatenated in the fasta but for raw reads it expects them to be individual to the sample which is why their path is stored in the columns "sra_file_column1".

For "SRA_file_location" the value "cloud" for cloud url raw reads or "local" for local raw reads can be added as the keyword SeqSender uses. Based on that keyword, SeqSender then expects the contents of SRA_file_column1, SRA_file_column2, etc. to be either a URL or a filepath to the raw read for that sample. You can continue to add raw read files by increasing the number for the column and SeqSender will automatically pick them up.
